Task 3/Task 3.2/Task 3.2/Task 3.2/DynamicArray.cs

```diff
@@ -0,0 +1,250 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_3._2
+{
+    public class DynamicArray<T> : IEnumerable<T>, ICloneable
+    {
+        private protected T[] _myArray { get; set; }
+
+        public int capacity { get; private set; } = 8;
+
+        public int length { get; private set; } = 0;
+
+        public DynamicArray()
+        {
+            _myArray = new T[capacity];
+        }
+
+        public DynamicArray(int value)
```

Comment:
`value` to me seems like it's a value that should be put in an array

Rename it to `capacity` so it's purpose is clear